src/styles/buy.css

```diff
@@ -0,0 +1,22 @@
+.buy__button {
```

Comment:
AS per BEM we need to put styles of BLOCK in separate file whereas the elements will go into same file

Reply:
Here buy is a block and so I have created a separate file buy.css , and since I don't need to add any styles on buy class directly so you just see it's elements buy__button here.